easybuild/easyconfigs/w/wxPython/wxPython-3.0.2.0-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,41 @@
+name = 'wxPython'
+version = '3.0.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.wxpython.org'
+description = """wxPython is a GUI toolkit for the Python programming language.
+ It allows Python programmers to create programs with a robust,
+ highly functional graphical user interface, simply and easily.
+ It is implemented as a Python extension module (native code)
+ that wraps the popular wxWidgets cross platform GUI library,
+ which is written in C++."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s-src-%(version)s.tar.bz2']
+patches = [
+    'wxPython_%(version)s_intel.patch',
+    'wxPython-%(version)s_fix-abs-overload.patch',
+]
+checksums = [
+    'd54129e5fbea4fb8091c87b2980760b72c22a386cb3b9dd2eebc928ef5e8df61',  # wxPython-src-3.0.2.0.tar.bz2
+    'fa7edb55e0c0ad18f636f9d9af7d2ee4cf3be439c78db5cae6d0ee47130f0ec6',  # wxPython_3.0.2.0_intel.patch
+    '7bc2d671af6cd79930275c9e59c42a7c79ee6d66329190ac13df690f5def4d94',  # wxPython-3.0.2.0_fix-abs-overload.patch
+]
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('libpng', '1.6.32'),
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('LibTIFF', '4.0.8'),
```

Comment:
@hajgato This one should definitely to bumped to `4.0.9` to avoid conflicts, see failing tests Travis CI